src/Schema/Events/Click.ts

```diff
@@ -1150,6 +1150,27 @@ export interface ClickedEditArtwork {
   label: string
 }
 
+/**
+ * A user clicks on "Edit Alert" on the list of alerts
+ *
+ * This schema describes events sent to Segment from [[ClickedEditAlert]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedEditAlert",
+ *    alert_id: "alert-id"
```

Comment:
I can't see the `alert_id` in the interface below - is that intentional?

Reply:
Good catch! I forgot to add it